### ISSUE TITLE
Unauthorized Access (Spaceship::UnauthorizedAccessError)

### ISSUE THREAD

**Issue by ronpelayo** [fastlane/fastlane#20072]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Unauthorized Access on select_team

##### Command executed
```
Spaceship::Tunes.client.teams.each do |team|
teamId = team['contentProvider']['contentProviderId']
selected_team = Spaceship::Tunes.select_team(team_id:teamId.to_s)
```

##### Complete output when running fastlane, including the stack trace and command used

> /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/client.rb:941:in `handle_error': Unauthorized Access (Spaceship::UnauthorizedAccessError)
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/client.rb:922:in `block in send_request'
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/client.rb:677:in `with_retry'
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/client.rb:918:in `send_request'
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/client.rb:762:in `request'
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/client.rb:165:in `team_id='
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/tunes/tunes_client.rb:87:in `select_team'
> 	from /Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/fastlane-2.204.3/spaceship/lib/spaceship/tunes/spaceship.rb:35:in `select_team'

<details>
  <pre> It was running fine yesterday but today I got that error.  </pre>
</details>

### Environment
<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 12.0.1                                      |
| Ruby                        | 3.0.2                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.30.1 (Apple Git-130)          |
| Installation Source         | ~/.rbenv/versions/3.0.2/bin/fastlane        |
| Host                        | macOS 12.0.1 (21A559)                       |
| Ruby Lib Dir                | ~/.rbenv/versions/3.0.2/lib                 |
| OpenSSL Version             | OpenSSL 1.1.1l  24 Aug 2021                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 13.1                                        |
| Swift Version               | 5.5.1                                       |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.204.3 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.5.0        |
| atomos                          | 0.1.3        |
| rexml                           | 3.2.5        |
| CFPropertyList                  | 3.0.4        |
| claide                          | 1.0.3        |
| colored2                        | 3.1.2        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.21.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| plist                           | 3.6.0        |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.8.0        |
| multipart-post                  | 2.0.0        |
| word_wrap                       | 1.0.0        |
| optparse                        | 0.1.1        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| artifactory                     | 3.0.15       |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| excon                           | 0.88.0       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.8.0        |
| unf_ext                         | 0.0.8        |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.4        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.2.0        |
| fastimage                       | 2.2.5        |
| gh_inspector                    | 1.1.3        |
| json                            | 2.5.1        |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| xcpretty-travis-formatter       | 1.0.1        |
| dotenv                          | 2.7.6        |
| bundler                         | 2.2.22       |
| simctl                          | 1.6.8        |
| jwt                             | 2.3.0        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.1        |
| representable                   | 3.1.1        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.2        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.1        |
| signet                          | 0.16.0       |
| googleauth                      | 1.1.0        |
| httpclient                      | 2.8.3        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.4.1        |
| google-apis-playcustomapp_v1    | 0.6.0        |
| google-apis-androidpublisher_v3 | 0.13.0       |
| google-cloud-env                | 1.5.0        |
| google-cloud-errors             | 1.2.0        |
| google-cloud-core               | 1.6.0        |
| google-apis-iamcredentials_v1   | 0.8.0        |
| google-apis-storage_v1          | 0.9.0        |
| rake                            | 13.0.3       |
| digest-crc                      | 0.6.4        |
| google-cloud-storage            | 1.34.1       |
| emoji_regex                     | 3.2.3        |
| jmespath                        | 1.4.0        |
| aws-partitions                  | 1.521.0      |
| aws-eventstream                 | 1.2.0        |
| aws-sigv4                       | 1.4.0        |
| aws-sdk-core                    | 3.121.5      |
| aws-sdk-kms                     | 1.50.0       |
| aws-sdk-s3                      | 1.104.0      |
| tsort                           | 0.1.0        |
| uri                             | 0.10.1       |
| set                             | 1.0.1        |
| forwardable                     | 1.3.2        |
| logger                          | 1.4.3        |
| pathname                        | 0.1.0        |
| shellwords                      | 0.1.0        |
| cgi                             | 0.2.0        |
| date                            | 3.1.0        |
| timeout                         | 0.1.1        |
| stringio                        | 3.0.0        |
| openssl                         | 2.2.0        |
| io-nonblock                     | 0.1.0        |
| ipaddr                          | 1.2.2        |
| io-wait                         | 0.1.0        |
| zlib                            | 1.1.0        |
| resolv                          | 0.2.0        |
| securerandom                    | 0.1.0        |
| digest                          | 3.0.0        |
| time                            | 0.1.0        |
| open-uri                        | 0.1.0        |
| mutex_m                         | 0.1.1        |
| net-protocol                    | 0.1.0        |
| ostruct                         | 0.3.1        |
| english                         | 0.7.1        |
| erb                             | 2.2.0        |
| strscan                         | 3.0.0        |
| abbrev                          | 0.1.0        |
| io-console                      | 0.5.7        |
| tempfile                        | 0.1.1        |
| delegate                        | 0.2.0        |
| fileutils                       | 1.5.0        |
| tmpdir                          | 0.1.2        |
| base64                          | 0.1.0        |
| singleton                       | 0.1.1        |
| net-http                        | 0.1.1        |
| open3                           | 0.1.1        |
| nkf                             | 0.1.0        |
| prettyprint                     | 0.1.0        |
| pp                              | 0.1.0        |
| find                            | 0.1.0        |
| yaml                            | 0.1.1        |
| psych                           | 3.3.0        |
</details>

*generated on:* **2022-03-17**
</details>

**Comment by brockw** [comment]
Yep also have this

**Comment by KangPilGyu** [comment]
I have the same issue

**Comment by thehellmaker** [comment]
Same issue

**Comment by cuongtv51** [comment]
Yep, I have same issue

**Comment by eli-stewart** [comment]
same issue

**Comment by ptrkstr** [comment]
Similar error.
```
INFO [2022-03-17 09:02:25.17]: Successfully loaded '/Users/XXX/git/XXX/fastlane/Deliverfile' 📄
WARN [2022-03-17 09:02:25.17]: No values defined in './fastlane/Deliverfile'
INFO [2022-03-17 09:02:25.17]: Login to App Store Connect (XXX)
Reading keychain entry, because either user or password were empty
Loading session from '/Users/XXX/.fastlane/spaceship/XXX/cookie'
Looking for App Store Connect Team with ID 117831961
Auth error received: 'Spaceship::UnauthorizedAccessError', 'Unauthorized Access'. Login in again then retrying after 3 seconds (remaining: 4)...
Loading session from '/Users/XXX/.fastlane/spaceship/XXX/cookie'
Auth error received: 'Spaceship::UnauthorizedAccessError', 'Unauthorized Access'. Login in again then retrying after 3 seconds (remaining: 3)...
Loading session from '/Users/XXX/.fastlane/spaceship/XXX/cookie'
Auth error received: 'Spaceship::UnauthorizedAccessError', 'Unauthorized Access'. Login in again then retrying after 3 seconds (remaining: 2)...
Loading session from '/Users/XXX/.fastlane/spaceship/XXX/cookie'
Auth error received: 'Spaceship::UnauthorizedAccessError', 'Unauthorized Access'. Login in again then retrying after 3 seconds (remaining: 1)...
Loading session from '/Users/XXX/.fastlane/spaceship/XXX/cookie'
```

**Comment by lowfreq2010** [comment]
we have the issue here, any thoughts about how to workaround?

**Comment by gooson** [comment]
same issue

**Comment by peter9teufel** [comment]
I am experiencing the same issue, pilot fails with UnauthorizedAccess Error

**Comment by ngocpd-1250** [comment]
same issue

**Comment by madearyadi** [comment]
same issue here

**Comment by Thangaraj-M** [comment]
facing same issue

**Comment by BobDG** [comment]
Same issue, I did update to macOS Monterey 12.3 yesterday, perhaps that's involved somehow?

**Comment by hmphu** [comment]
I'm facing same issue :( very frustrated

**Comment by henwilli** [comment]
same issue here

**Comment by ronpelayo** [comment]
> Same issue, I did update to macOS Monterey 12.3 yesterday, perhaps that's involved somehow?

I didn't do anything and it's working fine yesterday. Maybe apple updated their API?

**Comment by harryfb** [comment]
Same issue

**Comment by Bogdan-Belogurov** [comment]
Same issue here

**Comment by traktor74** [comment]
Same issue =(

**Comment by ltimmers** [comment]
Same here. Worked without a problem till yesterday (version 2.204.3). Not done any OS or other updates.

**Comment by niltsh** [comment]
Same issue, Yesterday it was OK.

And today I just updated my machine to 12.3.

**Comment by KeimyPlaza** [comment]
Same issue from today at 02:00 GMT+1

**Comment by einar-notland** [comment]
I get the same. When i log in to appstoreconnect, it seems like Apple are having some issues though, might be related.

![Screenshot 2022-03-17 at 14 06 56](https://user-images.githubusercontent.com/76111774/158814906-a182b398-daa0-4e46-8984-1ed28b2772f8.png)

**Comment by welrachid** [comment]
same here

**Comment by abarisic86** [comment]
Revert Fastlane to `2.176.0.`

**Comment by ramnac** [comment]
@abarisic86 reverting fastlane to 2.176.0 - did it work - curious ?

**Comment by BDProject** [comment]
Anybody able to confirm any work around for this issue? @fastlaneTeam this being looked into?

**Comment by litinskii** [comment]
Same error

**Comment by 27leaves** [comment]
Thank you @ronpelayo , this works as a quick fix!
```ruby
// Gemfile
# running with ruby 2.7.3
source "https://rubygems.org"

gem "fastlane", "2.176.0"
```
and then `bundle install`

**Comment by mazefest** [comment]
Same issue

**Comment by tuannvadrone** [comment]
Please add

gem "fastlane", "2.205.0"

in Gemfile

**Comment by ronpelayo** [comment]
Issue now fixed on fastlane `[2.205.0](https://github.com/fastlane/fastlane/pull/20083)`


New issue on new release. Error when running [select_team](https://github.com/fastlane/fastlane/issues/20086)

**Comment by tymcdowell** [comment]
> Issue now fixed on fastlane [2.205.0](https://github.com/fastlane/fastlane/pull/20083)
> 
> New issue on new release. Error when running [select_team](https://github.com/fastlane/fastlane/issues/20086)

Same issue here

**Comment by ecorreyu** [comment]
With version 2.205.1 it still crashes, any recommendations?
